packages/astro/src/default/components/MetaTags.astro

```diff
@@ -15,6 +15,10 @@ if (meta.image) {
   }
 }
 imageUrl ??= readLogoFile('logo', true);
+if (imageUrl?.endsWith('.svg')) {
+  console.warn(`Using a SVG open graph image "${imageUrl}". This is not supported by most social platforms. 
+  You should rather set "meta.image" to a raster image (PNG, WEBP).`);
+}
```

Comment:
By default `npm create tutorial` creates `public/logo.svg`, which is used here by default when `meta.image` is not defined. This means that this error message would be shown to all new projects with default settings.

Reply:
Yes I came to the same conclusion so that's why I think it would be reasonable to also add a "cover.webp" in "/public", and use that as the default
Having no image is a major penalty on social media and search engines imo, having at least something like "A tutorialkit lesson" with the framework logo would be totally acceptable and preferable to nothing.

Reply:
@sulco do you have any good images that we could use as default og-image in the created tutorials?